packages/ssr/src/utils/chunker.ts

```diff
@@ -7,15 +7,14 @@ function createChunkRegExp(chunkSize: number) {
 	return new RegExp('.{1,' + chunkSize + '}', 'g');
 }
 
-const MAX_CHUNK_SIZE = 3600;
+const MAX_CHUNK_SIZE = 1600;
```

Comment:
We need to remember to set this back to 3600 as this was only for testing.

Reply:
I was wondering about that!